com.unity.render-pipelines.high-definition/Editor/Lighting/LightmappingHDRP.cs

```diff
@@ -0,0 +1,156 @@
+using System;
+using JetBrains.Annotations;
+using UnityEngine;
+using UnityEngine.Experimental.Rendering;
+using UnityEngine.Rendering.HighDefinition;
+
+namespace UnityEditor.Rendering.HighDefinition
+{
+    /// <summary>
+    ///     Lightmapping utilities for HDRP
```

Comment:
Provides methods to help you with lightmapping in the High Definition Render Pipeline.